src/core/lib/iomgr/tcp_client_posix.cc

```diff
@@ -86,6 +100,41 @@ void do_tcp_client_global_init(void) {
   g_connection_shards = new std::vector<struct ConnectionShard>(num_shards);
 }
 
+int64_t event_engine_tcp_client_connect(
+    grpc_closure* on_connect, grpc_endpoint** endpoint,
+    const grpc_event_engine::experimental::EndpointConfig& config,
+    const grpc_resolved_address* addr, grpc_core::Timestamp deadline) {
+  auto resource_quota = reinterpret_cast<grpc_core::ResourceQuota*>(
+      config.GetVoidPointer(GRPC_ARG_RESOURCE_QUOTA));
```

Comment:
`config.GetObject<grpc_core::ResourceQuota>()`

Reply:
There's no GetObject method in EndpointConfig object. Its only present in the channel_args type.